src/frontend/src/hooks/GroupApplicationHooks.ts

```diff
@@ -15,7 +16,8 @@ export default function useGroupApplication(groupId: Guid) {
             setLoading(true);
 
             const response = await fetch(`${ApiConfig.API_URL}/groupapplication/${groupId}`, {
-                ...defaultRequestOptions
+                ...defaultRequestOptions,
+                ...{Authorization: `Bearer: ${useAuth().session?.access_token}`}
```

Comment:
we should store the `userAuth` hook data response in the state to avoid potentially leaking state

```js
const [ session, errors ] = useAuth();
```

Reply:
Yeah we should